MMM-pages.js

```diff
@@ -36,15 +33,15 @@ Module.register('MMM-pages', {
    * @param {number} x The dividend
    * @param {number} n The divisor
    */
-  mod: function (x, n) {
-    return ((x % n) + n) % n;
+  mod (x, n) {
+    return (x % n + n) % n;
```

Comment:
Is there a way to avoid removing the parens here or keeping the operation order unambiguous?

Modulus precedence is always something I never remember well, so this is the one area where I intentionally tried keeping that explicit.

Reply:
No problem. I disabled the rule which caused that change (`@stylistic/no-extra-parens`).